### PR TITLE
Minor bug fixes

### PR DESCRIPTION
**Pull request by meowsocute**
🔄 Updated `.gitignore`.
🛠 Created fallback component for upcoming pages in `$components/dev.todo.svelte.`
🔄 Replaced deprecated `Map` with new `SvelteMap`.
🐛 Fixed minor bugs in `/anime/+page.svelte` (Updated href URLs).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,5 @@ node_modules
 vite.config.js.timestamp-*
 vite.config.ts.timestamp-*
 stats.html
+package-lock.json
+pnpm-lock.yaml
```
Comment (moonlitgrace) [inline .gitignore:13]: These files are still being tracked. I think you've to clear git cache then commit again.

File: src/routes/anime/+page.svelte
Comment (moonlitgrace) [inline src/routes/anime/+page.svelte:1]: btw you dont have to worry bout mock datas, we will use backend api soon.

File: .gitignore (hunk 1 above)
Comment (baseplate-admin) [reply .gitignore:13]: Why is this in here?

This will break deoendabot

File: package.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
 		"@sveltejs/adapter-auto": "^3.2.2",
 		"@sveltejs/adapter-node": "^5.2.0",
 		"@sveltejs/adapter-static": "^3.0.2",
+		"@sveltejs/adapter-vercel": "^5.4.1",
```
Comment (baseplate-admin) [inline package.json:30]: This should not need to be added. Adapter-auto does the trick

Comment (meowsocute) [reply package.json:30]: Ok, i'm goonna remove e'm

File: src/lib/functions/map/set_all_to_value.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-export const set_all_to_value = <K, V>(map: Map<K, V>, value: V) => {
+export const set_all_to_value = <K, V>(map: SvelteMap<K, V>, value: V) => {
```
Comment (baseplate-admin) [inline src/lib/functions/map/set_all_to_value.ts:1]: Why is this a `SvelteMap`?

File: src/lib/stores/modals.svelte.ts [hunk 4]
```diff
@@ -1,10 +1,10 @@
-import { Map } from "svelte/reactivity";
+import {  SvelteMap } from "svelte/reactivity";
 import { set_all_to_value } from "$functions/map/set_all_to_value";
 
 const modals = ["search"] as const;
 type IModals = (typeof modals)[number];
 
-var state_map = $state(new Map<IModals, boolean>(modals.map((item) => [item, false])));
+var state_map = $state(new SvelteMap<IModals, boolean>(modals.map((item) => [item, false])));
```
Comment (baseplate-admin) [inline src/lib/stores/modals.svelte.ts:7]: This should be a `let`

File: src/routes/+page.svelte [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<script>
+	import DevTodo from "$components/dev.todo.svelte";
```
Comment (baseplate-admin) [inline src/routes/+page.svelte:2]: Why is this a component?

Just use a normal page?

**Review by baseplate-admin — CHANGES_REQUESTED** [review]
Hi, overall good job. Here are some minor picks i have

File: src/lib/functions/map/set_all_to_value.ts (hunk 3 above)
Comment (meowsocute) [reply src/lib/functions/map/set_all_to_value.ts:1]: Map deprecated

File: src/routes/+page.svelte (hunk 5 above)
Comment (meowsocute) [reply src/routes/+page.svelte:2]: we need this component as a template for todo pages. Refactored for those pages

File: src/lib/stores/modals.svelte.ts (hunk 4 above)
Comment (meowsocute) [reply src/lib/stores/modals.svelte.ts:7]: Its same as previous, even when I cloned

File: src/lib/functions/map/set_all_to_value.ts (hunk 3 above)
Comment (baseplate-admin) [reply src/lib/functions/map/set_all_to_value.ts:1]: I think there's no need for a reactive map in this case. It's just a plain JS function :)

File: src/lib/stores/modals.svelte.ts (hunk 4 above)
Comment (baseplate-admin) [reply src/lib/stores/modals.svelte.ts:7]: > Its same as previous, even when I cloned

Ah maybe i overlooked it. But please change it to `let`?

File: src/routes/+page.svelte (hunk 5 above)
Comment (baseplate-admin) [reply src/routes/+page.svelte:2]: > we need this component as a template for todo pages. Refactored for those pages

While it makes sense to me, i think the better approach is how we did it previously. Overall i have no good opinion on this. 

CC: @moonlitgrace

File: src/lib/functions/map/set_all_to_value.ts (hunk 3 above)
Comment (meowsocute) [reply src/lib/functions/map/set_all_to_value.ts:1]: image

File: src/routes/+page.svelte (hunk 5 above)
Comment (moonlitgrace) [reply src/routes/+page.svelte:2]: well @meowsocute can we keep this as showing error page (like before)? cause we dont make this live without required pages.
what do you think @baseplate-admin

File: src/lib/functions/map/set_all_to_value.ts (hunk 3 above)
Comment (meowsocute) [reply src/lib/functions/map/set_all_to_value.ts:1]: IDK, the suggested it.... U r right anyways

File: src/lib/stores/modals.svelte.ts (hunk 4 above)
Comment (meowsocute) [reply src/lib/stores/modals.svelte.ts]: Changes made...

File: .gitignore (hunk 1 above)
Comment (meowsocute) [reply .gitignore:13]: removed in new commit

File: src/routes/+page.svelte (hunk 5 above)
Comment (meowsocute) [reply src/routes/+page.svelte:2]: I made it purpose fully, if anybody coming like me won't have idea it it shows an error page. And it remember us to  make those pages

Comment (moonlitgrace) [reply src/routes/+page.svelte:2]: > I made it purpose fully, if anybody coming like me won't have idea why it shows an error page.

I get it, but looking at `routes` can tell which pages are up and not.
and homepage design is under work, so honestly I dont see the point. can you revert it?

File: src/lib/stores/modals.svelte.ts [hunk 6]
```diff
@@ -1,10 +1,10 @@
-import { Map } from "svelte/reactivity";
+import {  SvelteMap } from "svelte/reactivity";
```
Comment (moonlitgrace) [inline src/lib/stores/modals.svelte.ts:1]: I think you forgot to remove this.
just build-in `Map` is good :)

File: src/routes/+page.svelte (hunk 5 above)
Comment (meowsocute) [reply src/routes/+page.svelte:2]: Yeh

File: src/lib/stores/modals.svelte.ts (hunk 6 above)
Comment (meowsocute) [reply src/lib/stores/modals.svelte.ts]: Should I replace `SvelteMap` with deprecated `Map`, Some how it is not even loading the mock data for me

Comment (meowsocute) [reply src/lib/stores/modals.svelte.ts]: Should I replace SvelteMap with deprecated Map, Some how it is not even loading the mock data for me

File: src/lib/stores/modals.svelte.ts [hunk 7]
```diff
@@ -1,10 +1,10 @@
-import {  SvelteMap } from "svelte/reactivity";
+import { Map } from "svelte/reactivity";
```
Comment (moonlitgrace) [inline src/lib/stores/modals.svelte.ts:1]: we can just remove this line

File: src/lib/stores/modals.svelte.ts (hunk 6 above)
Comment (moonlitgrace) [reply src/lib/stores/modals.svelte.ts]: No? we can use javascript build-in Map function

Comment (baseplate-admin) [reply src/lib/stores/modals.svelte.ts]: In this case @meowsocute  is correct. This should be `SvelteMap`

File: src/lib/stores/modals.svelte.ts (hunk 7 above)
Comment (baseplate-admin) [reply src/lib/stores/modals.svelte.ts]: I dont think this is the right approach.

**Comment by meowsocute** [comment]
I'm soo happy to get my first contribution on a large scaled project, 
Thank You @baseplate-admin @moonlitgrace and other contributers. 🎉

**Pull request merged**